py/server/tests/test_pandas.py

```diff
@@ -11,7 +11,7 @@
 from deephaven import dtypes, new_table, DHError
 from deephaven.column import byte_col, char_col, short_col, bool_col, int_col, long_col, float_col, double_col, \
     string_col, datetime_col, pyobj_col, jobj_col
-from deephaven.constants import NULL_LONG, NULL_SHORT, NULL_INT
+from deephaven.constants import NULL_LONG, NULL_SHORT, NULL_INT, NULL_BYTE as NB
```

Comment:
Why the `as NB`?

Reply:
one of the test cases expects the Java constant NULL_BYTE in the formula, this is to work around the naming conflicts between Python and Java. A related ticket is #2306.